pkgs/by-name/rm/rmenu/package.nix

```diff
@@ -65,14 +65,17 @@ rustPlatform.buildRustPackage rec {
     # fix config and theme
     mkdir -p $out/share/rmenu
     cp -vf $src/rmenu/public/config.yaml $out/share/rmenu/config.yaml
-    sed -i "s@~\/\.config\/rmenu\/themes@$out\/themes@g" $out/share/rmenu/config.yaml
-    sed -i "s@~\/\.config\/rmenu@$out\/plugins@g" $out/share/rmenu/config.yaml
+    sed -i "s@~\/\.config\/rmenu@$out@g" $out/share/rmenu/config.yaml
```

Comment:
Instead of sed you can use:
```
substituteInPlace $out/share/rmenu/config.yaml \
  --replace '~/.config/rmenu' '$out'
```

Reply:
Double quotes for $out to be resolved correctly, but this is indeed more readable, ty